lib/src/backend/test_platform.dart

```diff
@@ -92,3 +85,35 @@ class TestPlatform {
 
   String toString() => name;
 }
+
+List<TestPlatform> _all = [
```

Comment:
`final`. Also, put a type argument on the array – so the reified type is correct `<TestPlatform>[...`

Also `_all` is a bit too vague.